redis/src/streams.rs

```diff
@@ -510,6 +570,27 @@ impl FromRedisValue for StreamClaimReply {
     }
 }
 
+impl FromRedisValue for StreamAutoClaimReply {
+    fn from_redis_value(v: &Value) -> RedisResult<Self> {
+        let (start_stream_id, rows, deleted_ids): (
+            String,
+            Vec<HashMap<String, HashMap<String, Value>>>,
+            Vec<String>,
+        ) = from_redis_value(v)?;
```

Comment:
When just_id option is provided, this fails. Should we:

1. Provide empty map
	1. Parse rows as `Vec<Value>`,
	2. Check if Value can be parsed as `HashMap<_, _>`
	3. Parse as `HashMap<>` if so, else parse as `Vec<string>` then create `Vec<StreamId>` 
2. Add warning to `StreamAutoClaimReply` that it would fail when just ids is provided

option 1 seems what I would expect as API consumer

Reply:
I think empty hash maps sounds less than ideal. Maybe we should just add an additional type to be used for passing `JUSTID`?

Reply:
In my PR #1169 (didn't know this one existed at the time) I took the approach of having the FromRedisValue method handle both kinds of responses and using an empty hash.   I had originally done a separate type, but that can get confusing to the user.  Thus a single type was overall simpler and abstracts away the differences so they can just use one contract to access the response.  IMHO this makes coding easier.  And a helper function can be added to the StreamAutoClaimReply struct to be able to easily get to the IDs if desired.

Also my PR handles differences between Redis 6 and redis 7 responses from this  command. (they are different)